test/built-ins/Math/pow/applying-the-exp-operator_A10.js

```diff
@@ -19,8 +19,8 @@ var basenum = 6;
 
 for (var i = 0; i < basenum; i++)
 {
```

Comment:
Are you intending to enforce a brace style, or is this mostly just about whitespace?

Reply:
it should be whitespace, the main motivator is avoiding unnecessary trailing spaces and mixed tabs&spaces. Some other stylish changes are collateral.

Reply:
> Are you intending to enforce a brace style, or is this mostly just about whitespace?

I have a whole other processor ready for that. I'm glad you asked, because now I feel a greater sense of empowerment to go for it :D